FirebaseAuth/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [added] Added reCAPTCHA enterprise support before sending SMS. This includes
```

Comment:
Can this be reworded to be more user-centric? Something like "Phone  and Multi-factor Authentication can now use reCAPTCHA enterprise for verification, as an alternative to SMS."

Reply:
Updated to `Added reCAPTCHA Enterprise support for app verification during phone authentication for Firebase Authentication` to align with [Android](https://firebase.google.com/support/release-notes/android#auth_v23-1-0) and [Web releases](https://firebase.google.com/support/release-notes/js#authentication).